conference/react-js/src/components/User-menu-filter/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UserMenu from '../User-menu';
+import {
+  baseUrl,
+  settings,
+  manageUser,
+  talks,
+  myTalks,
+  myInfo,
+} from '../../constants/route-url';
+import {
+  speaker,
+  admin,
+} from '../../constants/roles';
+
+const UserMenuFilter = ({ roles }) => {
+  let data = [
+    { title: 'Conferences', link: baseUrl },
+    { title: 'Talks', link: talks },
+  ];
+  const speakerRole = roles.indexOf(speaker);
+  const adminRole = roles.indexOf(admin);
+  if (speakerRole > -1) {
+    data = [...data, { title: 'My talks', link: myTalks },
+      { title: 'My info', link: myInfo }];
+  }
+  if (adminRole > -1) {
```

Comment:
is there is a possibility for scenario when `adminRole` and `speakerRole` indexes are both > -1.
if no, we can put `if`s together in `if-else-if`

Reply:
Change it in if-else